app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -9,6 +9,11 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
     $scope.yearMoinsUn = moment($scope.situation.dateDeValeur).subtract('years', 1).format('YYYY');
     $scope.currentMonth = moment($scope.situation.dateDeValeur).format('MMMM YYYY');
 
+    // Pour les Auto-entrepreneurs
+    $scope.onGoingLabel = 'J\'aurai un chiffre d\'affaires non nul en ' + $scope.currentMonth;
```

Comment:
I first try double quotes, but it turned a JSHINT test to red, hence my last fixup commit